src/workerd/api/tests/websocket-client-error-sidecar.js

```diff
@@ -68,8 +68,14 @@ function sendMessage(socket, message) {
   socket.write(frame);
 }
 
-// Get the port from environment variable
+// Get the port and host from environment variables
 const port = process.env.BIG_MESSAGE_SERVER_PORT || 9000;
+const host = process.env.SIDECAR_HOST || 'localhost';
```

Comment:
Why is this optional?

Reply:
Dunno. The existing code was like this, but it's gone now.